### ISSUE TITLE
Latest commit doesn't build on JitPack

### ISSUE THREAD

**Issue by Leland-Takamine** [pedroSG94/rtmp-rtsp-stream-client-java#17]
Trying to depend on the latest commit version:

```
compile 'com.github.pedroSG94.rtmp-rtsp-stream-client-java:builder:9c15e72d7e6c063f99190f4c7ccbc285ffc4cb09'
```

But it looks like the build is [failing on jitpack](https://jitpack.io/com/github/pedroSG94/rtmp-rtsp-stream-client-java/9c15e72d7e6c063f99190f4c7ccbc285ffc4cb09/build.log).

**Comment by pedroSG94** [comment]
The last build on jitpack is:
```
allprojects {
  repositories {
    maven { url 'https://jitpack.io' }
  }
}
dependencies {
  compile 'com.github.pedroSG94.rtmp-rtsp-stream-client-java:builder:1.0.6'
}
```
Last commit is not uploaded at the moment. I will upload this week.

**Comment by Leland-Takamine** [comment]
JitPack has a feature that allows you to depend on a repo's specific commit. I've opened a PR to fix the issue: #18.
